### ISSUE TITLE
Feature Request: Fix URL in npmjs

### ISSUE THREAD

**Issue by BrodaNoel** [fnando/i18n#108]
## Description

After reading this issue https://github.com/fnando/i18n/issues/107, I would suggest to update the project URL in https://www.npmjs.com/package/i18n-js

It is currently pointing to https://github.com/fnando/i18n, instead of https://github.com/fnando/i18n-js

## Describe the solution
Update URL

## Alternatives you considered

n/a

## Additional context

I'm quite sure that those 120.000 downloads in https://www.npmjs.com/package/i18n-js are because people is opening the project URL in npmjs, pointing to this wrong project, and after not seeing any migration file, they just quit the migration.

<img width="426" alt="Screen Shot 2024-11-18 at 21 45 46" src="https://github.com/user-attachments/assets/3ef50426-1a67-4d4a-930f-1742e27a1358">

**Comment by fnando** [comment]
@fnando/i18n is the javascript package, while https://github.com/fnando/i18n-js hosts the CLI code. As of today, they're independent and one doesn't require the other.

The reason that happens today is because the original version of i18n-js was too rails focused, while some people wanted to use i18n-js on non-rails environments.

tl;dr: `npm i @fnando/i18n` is the way to go today, while `gem install i18n-js` will install the CLI and give you some niceties on your workflow.

**Comment by BrodaNoel** [comment]
How is that possible? I have `i18n-js` in my project since 6 years ago.

<img width="438" alt="Screen Shot 2024-11-18 at 21 54 35" src="https://github.com/user-attachments/assets/5ea0190e-7812-4619-a111-ff2dfe23599b">

<img width="310" alt="Screen Shot 2024-11-18 at 21 54 58" src="https://github.com/user-attachments/assets/1010f0ab-c15a-4cae-9e3d-af476e5c69cf">

**Comment by fnando** [comment]
The change happened on v4, released about two years ago.

**Comment by BrodaNoel** [comment]
Just to clarify, I'm using `i18n-js` in a React Native project, using the `i18n.t` method.
